dev/com.ibm.ws.microprofile.metrics.2.0.monitor/src/com/ibm/ws/microprofile/metrics/monitor/MappingTable.java

```diff
@@ -63,13 +62,6 @@ private MappingTable() {
         };
 		mappingTable.put("WebSphere:type=ServletStats,name=*", servletTable);
 		
-		String[][] restfulTable = new String[][] {
-        	{ "RESTful.request.total", "Total Request", "RESTful.request.total.description", COUNTER, MetricUnits.NONE, "RequestCount", null, RESTFUL_TAG_NAME },
-        	{ "RESTful.responseTime.total", "Total Response Time", "RESTful.responseTime.total.description", GAUGE, MetricUnits.NANOSECONDS, "ResponseTimeDetails", "total", RESTFUL_TAG_NAME }
-        };
-		mappingTable.put("WebSphere:type=RESTful_Stats,name=*", restfulTable);
-
-
```

Comment:
With removing the mapping stuff, are you planning to add that back in in a separate PR?  Or is David?

Reply:
Yes,  it is part of Davids PR:  https://github.com/OpenLiberty/open-liberty/pull/10518